test/steps/CommonSteps.scala

```diff
@@ -38,18 +38,16 @@ object Injector {
 
 object CommonSteps extends PlaySpec with GuiceOneServerPerSuite with BeforeAndAfterAll with MockitoSugar with Injecting {
 
-  case class ProjectBean(id: String, name: String, repositoryUrl: String, stableBranch: String, featuresRootPath: String)
-
 
   var response: Future[Result] = _
 
   var projects: Map[String, Project] = _
 
-  val projectService = new ComponentService()
+  val projectService = Injector.inject[ProjectService]
 
   implicit val db = Injector.inject[Database]
 
-  override def fakeApplication(): Application = new GuiceApplicationBuilder().overrides(bind[ComponentService].toInstance(projectService)).in(Mode.Test).build()
+  override def fakeApplication(): Application = new GuiceApplicationBuilder().overrides(bind[ProjectService].toInstance(projectService)).in(Mode.Test).build()
```

Comment:
Remove `.overrides(bind[ProjectService].toInstance(projectService))` as projectService  is not a mock